crates/aranya-model/src/model.rs

```diff
@@ -366,6 +373,30 @@ where
         Ok(sink.effects)
     }
 
+    /// Remove a graph from a client
+    fn remove_graph(
+        &mut self,
+        proxy_id: Self::GraphId,
+        client_proxy_id: Self::ClientId,
+    ) -> Result<(), ModelError> {
+        let Some(storage_id) = self.storage_ids.remove(&proxy_id.into()) else {
+            return Err(ModelError::GraphNotFound);
+        };
+
+        let mut state = self
+            .clients
+            .get_mut(&client_proxy_id.into())
+            .ok_or(ModelError::ClientNotFound)?
+            .state
+            .borrow_mut();
```

Comment:
Nit: use infallible `get_mut()` when you have a `&mut RefCell`. (I see existing code was the same)

```suggestion
            .get_mut();
```